integrationTests/runner/run.js

```diff
@@ -1,4 +1,5 @@
 const fs = require('fs');
+// eslint-disable-next-line import/no-unresolved -- ignore unresolved build artifact
```

Comment:
sorta surprised this is needed

Reply:
I had the lint error when the build folder is absent on my local machine.
Can still reproduce it with `$ git clean -xfd && yarn install && yarn run lint`

Reply:
right, we need to run a build before the lint run. meh, ignoring is fine